client/me/get-apps/index.jsx

```diff
@@ -1,82 +1,104 @@
+    <SectionHeader label={ translate( 'Desktop apps' ) } />
+    <Card className="get-apps__desktop">
 
-					<section className="get-apps__app">
-						<div className="get-apps__app-icon">
-							<img src="/calypso/images/me/get-apps-osx.svg" alt={ this.translate( 'Mac' ) } width="96" height="96" />
-						</div>
-						<p><strong>{ this.translate( 'Mac' ) }</strong><br />
-						{ this.translate( 'Requires 10.11 or newer' ) }</p>
-						<Button href="https://apps.wordpress.com/d/osx?ref=getapps">{ this.translate( 'Download' ) }</Button>
-					</section>
+      <section className="get-apps__app">
+        <div className="get-apps__app-icon">
+          <img src="/calypso/images/me/get-apps-osx.svg" alt={ translate( 'Mac' ) } width="96" height="96" />
+        </div>
+        <p><strong>{ translate( 'Mac' ) }</strong><br />
+        { translate( 'Requires 10.11 or newer' ) }</p>
+        <Button href="https://apps.wordpress.com/d/osx?ref=getapps">{ translate( 'Download' ) }</Button>
+      </section>
 
-					<section className="get-apps__app">
-						<div className="get-apps__app-icon">
-							<img src="/calypso/images/me/get-apps-windows.svg" alt={ this.translate( 'Windows' ) } width="96" height="96" />
-						</div>
-						<p><strong>{ this.translate( 'Windows' ) }</strong><br />
-						{ this.translate( 'Requires 7 or newer' ) }</p>
-						<Button href="https://apps.wordpress.com/d/windows?ref=getapps">{ this.translate( 'Download' ) }</Button>
-					</section>
+      <section className="get-apps__app">
+        <div className="get-apps__app-icon">
+          <img src="/calypso/images/me/get-apps-windows.svg" alt={ translate( 'Windows' ) } width="96" height="96" />
+        </div>
+        <p><strong>{ translate( 'Windows' ) }</strong><br />
+        { translate( 'Requires 7 or newer' ) }</p>
+        <Button href="https://apps.wordpress.com/d/windows?ref=getapps">{ translate( 'Download' ) }</Button>
+      </section>
 
-					<section className="get-apps__app">
-						<div className="get-apps__app-icon">
-							<img src="/calypso/images/me/get-apps-linux.svg" alt={ this.translate( 'Linux' ) } width="96" height="96" />
-						</div>
-						<p><strong>{ this.translate( 'Linux' ) }</strong><br />
-						{ this.translate( 'Choose your distribution' ) }</p>
-						<Button href="https://apps.wordpress.com/d/linux?ref=getapps">{ this.translate( '.TAR.GZ' ) }</Button> <Button href="https://apps.wordpress.com/d/linux-deb?ref=getapps">{ this.translate( '.DEB' ) }</Button>
-					</section>
+      <section className="get-apps__app">
+        <div className="get-apps__app-icon">
+          <img src="/calypso/images/me/get-apps-linux.svg" alt={ translate( 'Linux' ) } width="96" height="96" />
+        </div>
+        <p><strong>{ translate( 'Linux' ) }</strong><br />
+        { translate( 'Choose your distribution' ) }</p>
+        <Button
+          href="https://apps.wordpress.com/d/linux?ref=getapps"
+        >
+          { translate( '.TAR.GZ' ) }
```

Comment:
Again, I know you didn't change this here, but it seems like an odd thing to translate…

@akirk can you comment?